intellij-plugin-structure/structure-edu/src/main/kotlin/com/jetbrains/plugin/structure/edu/EduPlugin.kt

```diff
@@ -22,7 +22,8 @@ data class EduPlugin(
   override val thirdPartyDependencies: List<ThirdPartyDependency> = emptyList(),
   val descriptorVersion: Int? = null,
   val language: String? = null,
-  val programmingLanguage: String? = null,
```

Comment:
Shouldn't we keep this field for backwards compatibility with old EduPlugin versions?

Reply:
I think no. This is an internal object we use after plugin descriptor was parsed. For backward compatibility we still have old field in `EduPluginDescriptor` which is used while plugin descriptor is been parsed. 

I also added a separate test to check that backward compatibility is working correctly